scripts/src/main/resources/scripts/command/cobigen

```diff
@@ -32,13 +32,13 @@ function doSetup() {
 # $@: optional CobiGen arguments
 function doRunCLI() {
   local cobigen_cmd="cobigen"
-  doEcho "Running: ${cobigen_cmd} ${@}"
+  doEcho "Running: ${cobigen_cmd} ${*}"
   if doIsQuiet
   then
-    "${cobigen_cmd}" "${@}"
+    "${cobigen_cmd}" "${*}"
     exit ${?}
   else
-    "${cobigen_cmd}" "${@} --verbose"
+    "${cobigen_cmd}" "${*} --verbose"
```

Comment:
Similar here as above. But you also need to remove the "--verbose" from within the quote. The line should be
"${cobigen_cmd}" "${@}" --verbose